test/unit/autoNumeric.spec.js

```diff
@@ -4503,6 +4574,30 @@ describe('Instantiated autoNumeric functions', () => {
             expect(aNInput.getNumericString()).toEqual('0');
         });
 
+        it('should return number from `emptyInputBehavior` setting as the default value', () => {
```

Comment:
Note to self: I need to ditch the DOM element creation in each test and use `beforeEach`/`afterEach`..